config/bidderinfo.go

```diff
@@ -0,0 +1,88 @@
+package config
```

Comment:
Moved from the adapters package, with json serialization hacks removed, comments added, and refactored LoadBidderInfoFromDisk for easier testing.